client/state/automated-transfer/reducer.js

```diff
@@ -34,9 +35,13 @@ export const status = ( state = null, action ) =>
 		state
 	);
 
+export const fetchingStatus = ( state = [], action ) =>
+	REQUEST_STATUS === action.type ? true : state;
```

Comment:
This means that once the `fetchingStatus` is set to `true`, it's forever `true`. However, in case when the transfer becomes `COMPLETE`, `ERROR` or the request returns HTTP 404 error, we stop the polling loop.

In that case, shouldn't the `fetchingStatus` should become `false` again? If we then dispatch `fetchAutomatedTransferStatus` again, the polling loop should restart.

I'm pretty sure we should reset the `fetchingStatus` in case of 404 error (that also happens when no transfer exists yet). Not so sure about `COMPLETE` and `ERROR` -- these states mean that the transfer has finished, the status is "final" and no further fetching/polling is necessary.